e2e/app/modal/delete-runtime.modal.ts

```diff
@@ -0,0 +1,29 @@
+import { Page } from 'puppeteer';
+import { LinkText } from 'app/text-labels';
+import Modal from './modal';
+import { waitForText } from 'utils/waits-utils';
+
+const modalTitle = 'Delete Runtime';
+
+export default class DeleteRuntimeModal extends Modal {
+  // constructor(page: Page, xpath?: string) {
+  //   super(page, xpath);
+  constructor(page: Page, opts?: { xpath?: string; modalIndex?: number }) {
+    super(page, opts);
+  }
+
+  async isLoaded(): Promise<boolean> {
+    await waitForText(this.page, modalTitle, { container: this });
+    return true;
+  }
```

Comment:
Is `Delete` button enabled when modal is open? If it is, we should have `.waitUntilEnabled()` in `isLoaded`. And `await waitWhileLoading(this.page);`.